src/modules/mavlink/mavlink_main.h

```diff
@@ -430,9 +430,7 @@ class Mavlink : public ModuleParams
 	/**
 	 * Get the receive status of this MAVLink link
 	 */
-	telemetry_status_s	&get_telemetry_status() { return _tstatus; }
-	void                    lock_telemetry_status() { pthread_mutex_lock(&_telemetry_status_mutex); }
-	void                    unlock_telemetry_status() { pthread_mutex_unlock(&_telemetry_status_mutex); }
```

Comment:
`_telemetry_status_mutex` can be removed as well.

Reply:
Done.